clear-phi.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+psql -h localhost -U pulse -f clear-phi.sql pulse
```

Comment:
Can you make this take command line arguments for the host and user? The reset.sh script does something similar.